README.md

```diff
@@ -83,7 +83,8 @@ end
 First, you'll need to build the test database.
 
 ```elixir
-MIX_ENV=test mix ecto.reset
+docker-compose up
```

Comment:
I'd prefer not to have this directly in the README like this as it implies this is a required step.  I would be open to leaving a note here that speaks to "If you prefer to use docker, then do these steps..." type thing instead.